src/views/Contact.vue

```diff
@@ -1,9 +1,9 @@
 <template>
 	<div
-		class="bg-gray-100 flex items-center"
```

Comment:
The contact page seems different in the design. Could you please fix it?

![image](https://user-images.githubusercontent.com/27869979/98367128-67195100-2035-11eb-9def-88da0b7a3d3f.png)

![image](https://user-images.githubusercontent.com/27869979/98367215-89ab6a00-2035-11eb-8fbf-284c24df9ecb.png)